app/src/main/java/de/danoeh/antennapod/fragment/ItunesSearchFragment.java

```diff
@@ -24,6 +25,7 @@
 import org.json.JSONObject;
 
 import java.io.IOException;
+import java.io.InterruptedIOException;
```

Comment:
Nice, thanks for your PR. Please remove this unused import. Looks like it is a leftover from your experiments.